assembly-hadoop2.xml

```diff
@@ -156,6 +156,7 @@
                 <include>*:hadoop-common:*</include>
                 <include>*:commons-configuration:*</include>
                 <include>*:hadoop-auth:*</include>
+                <include>*:shifu-spark-eval:*</include>
```

Comment:
can we make it not in shifu core? I think make it as independent module and not included here